src/status_im/contexts/wallet/swap/set_spending_cap/view.cljs

```diff
@@ -228,9 +228,24 @@
 
 (defn- footer
   []
-  [rn/view {:style {:margin-bottom -10}}
-   [transaction-details]
-   [slide-button]])
+  (let [provider (rf/sub [:wallet/swap-proposal-provider])
+        theme    (quo.theme/use-theme)
+        on-press (rn/use-callback #(rf/dispatch [:open-url (:terms-and-conditions-url provider)])
```

Comment:
Quick question: We have a provider check to display the full name, but we don't do the same for the T&C URL.

Reply:
This may be a problem but only in development, I will add the check 👍

Reply:
Updated :)